### ISSUE TITLE
Duplicate type name after upgrade from 4.1.7 to 4.1.8

### ISSUE THREAD

**Issue by scmorse** [mtth/avsc#71]
First off, thanks for your work on this lib.

I don't know this code, so I'm not sure what could have caused this, but after upgrading from 4.1.7 to 4.1.8, I get this error when I try to assemble my schemas from an ivdl file:

```
Error: duplicate type name: tenable.asset.mgmt.v1.Asset
    at RecordType.Type (/service_asset_mart/node_modules/avsc/lib/types.js:180:17)
    at new RecordType (/service_asset_mart/node_modules/avsc/lib/types.js:1808:8)
    at /service_asset_mart/node_modules/avsc/lib/types.js:135:14
    at Object.createType (/service_asset_mart/node_modules/avsc/lib/types.js:136:7)
    at Object.opts.typeHook (/service_asset_mart/node_modules/avsc/lib/protocols.js:141:15)
    at createType (/service_asset_mart/node_modules/avsc/lib/types.js:79:37)
    at /service_asset_mart/node_modules/avsc/lib/types.js:850:51
    at Array.map (native)
    at UnwrappedUnionType.UnionType (/service_asset_mart/node_modules/avsc/lib/types.js:850:23)
    at new UnwrappedUnionType (/service_asset_mart/node_modules/avsc/lib/types.js:905:13)
```

The file only has one declaration of the `tenable.asset.mgmt.v1.Asset`, which leads me to believe there was a bug introduced in 4.1.8.

**Comment by mtth** [comment]
Thanks for reporting this. Any chance you could attach your IDL files (or an excerpt that reproduces the bug)?

**Comment by mtth** [comment]
I think I was able to identify the issue; I reverted the corresponding changes for now and released a new version (`4.1.9`). Let me know if you are still running into errors.

**Comment by scmorse** [comment]
Thanks, I do not see any errors with 4.1.9, so going to close this issue.
